src/store/mod.rs

```diff
@@ -81,18 +82,30 @@ where
     async fn interface_props(&self, interface: &str) -> Result<Vec<StoredProp>, Self::Err>;
 }
 
-/// Data structure used to return stored properties by a database implementing the AstarteDatabase
+/// Data structure used to return stored properties by a database implementing the [`PropertyStore`]
 /// trait.
 #[derive(Debug, Clone, Copy, PartialOrd)]
 pub struct StoredProp<S = String, V = AstarteType> {
+    /// Interface name of the property
     pub interface: S,
+    /// Path of the property's mapping.
     pub path: S,
+    /// Value of the property.
     pub value: V,
+    /// Major version of the interface.
+    ///
+    /// This is important to check if a stored property is compatible with the current interface
+    /// version.
     pub interface_major: i32,
+    /// Ownership of the property.
+    ///
+    /// If its [`Ownership::Device`] the property was sent from the device to Astarte. Instead, if
```

Comment:
Missing apostrophe (it's). It could also be rendered as "When [`Ownership::Device`], the property [...]. Instead, when [...]", perhaps.